### ISSUE TITLE
通过fate-9999和fate-10000.yaml文件创建好cluster后，Kubefate cluster describe XXX命令显示 the server could not find the requested resource

### ISSUE THREAD

**Issue by jeanP-zhang** [FederatedAI/KubeFATE#563]
**What deployment mode you are use?  **
2. Kuberentes.

**What KubeFATE and FATE version you are using?  
KubeFateV1.7.2
**MUST** Please state the KubeFATE and FATE version you found the issue

**What OS you are using for docker-compse or Kubernetes? Please also clear the version of OS.  **
  Ubuntu20.04 LTS

**Desktop (please complete the following information):**

**To Reproduce**

Clear how to reproduce your problem.

**What happen?**
 
通过fate-9999和fate-10000.yaml文件创建好cluster后，Kubefate cluster describe 命令显示 the server could not find the requested resource，但是fate网页可以打开。
 
Clear the unexpected response.

**Screenshots**

![image](https://user-images.githubusercontent.com/54132884/159423394-4d8f24e7-6afe-40ea-aa38-951a25d2281f.png)

**Additional context**

**Comment by JingChen23** [comment]
https://github.com/FederatedAI/KubeFATE/pull/553
We have made a fix for that issue and the fix will be released by Kubefate V1.8.0, as your version is V1.7.2, so seeing the issue should be expected.
CC @owlet42 in case I'm wrong.

**Comment by jeanP-zhang** [comment]
> We have made a fix for that issue (#553) and the fix will be released by Kubefate V1.8.0, as your version is V1.7.2, so seeing the issue should be expected. CC @owlet42 in case I'm wrong.

谢谢

**Comment by JingChen23** [comment]
```
[root@sc2-rdops-vm05-dhcp-173-64 kubefate]# kubefate cluster list
UUID                                	NAME      	NAMESPACE 	REVISION	STATUS 	CHART	ChartVERSION	AGE
a3185b0f-def9-4c38-b3ca-9839b71a2cf1	fate-9999 	fate-9999 	1       	Running	fate 	v1.7.2      	11h
80d81890-aa78-4a52-a718-e042a7d2945e	fate-10000	fate-10000	1       	Running	fate 	v1.7.2      	11h

[root@sc2-rdops-vm05-dhcp-173-64 kubefate]# kubefate cluster describe a3185b0f-def9-4c38-b3ca-9839b71a2cf1
UUID        	a3185b0f-def9-4c38-b3ca-9839b71a2cf1
Name        	fate-9999
NameSpace   	fate-9999
ChartName   	fate
ChartVersion	v1.7.2
Revision    	1
Age         	11h
Status      	Running
Spec        	backend: eggroll
            	chartName: fate
            	chartVersion: v1.7.2
            	imagePullSecrets:
```

Actually I cannot reproduce the issue.

I guess the reason is:
You are not using the released 1.7.2 package (https://github.com/FederatedAI/KubeFATE/releases/download/v1.7.2/kubefate-k8s-v1.7.2.tar.gz). You are using the code on the master branch, and unfortunately you pull the code before the fix had been done.

Close the issue, thanks.
